client/resource_group/controller/controller.go

```diff
@@ -433,6 +435,11 @@ func (c *ResourceGroupsController) tryGetResourceGroup(ctx context.Context, name
 		return nil, err
 	}
 	if group == nil {
+		// If the resource group does not exist and it's not the default resource group and fallback is true,
+		// try again with the default resource group.
+		if fallback && name != defaultResourceGroupName {
```

Comment:
Here path is after calling gRPC, there may many gRPC calls according to the request. maybe we need to rewrite it as default in the topper level.

Reply:
I modified the logic here:

- When fallback is triggered, the default group controller will be used as an alias.
- When the actual group creation event is watched, it will replace the default.
- The cleanup that runs every 5 minutes will delete the fallback group controller.